### ISSUE TITLE
Slate v0 is not reloading browser after trying Slate V1

### ISSUE THREAD

**Issue by radmedov** [Shopify/slate#906]
**### Problem**
I was trying to use Slate V1 on an existing project then I was back to Salte v0.
But the browsersync doesn't reload the browser anymore.

Any idea why?
P.S. I've removed mkcert, yarn and killed HSTS in chrome already.  

Thank you.

**Comment by t-kelly** [comment]
@radmedov can you try viewing you Slate v0 project in a Chrome incognito window? Pretty sure this is because Slate v1 uses `https://localhost` and not `http`, which can cause Chrome to default to using `https` in the future.

**Comment by radmedov** [comment]
@t-kelly It doesn't work in an incognito mode as well. It's also not working in any browser.
I'm getting in the Chrome console:

> This site does not have a valid SSL certificate! Without SSL, your site's and visitors' data is vulnerable to theft and tampering. Get a valid SSL certificate before releasing your website to the public. 

Firefox's console:

> This site makes use of a SHA-1 Certificate; it’s recommended you use certificates with signature algorithms that use hash functions stronger than SHA-1.

> 
Strict-Transport-Security: The connection to the site is untrustworthy, so the specified header was ignored.

Mkcert certificate was removed. The browser uses the certificate from the node-modules folder of the project.
I've tried to build a new slate v0 project with just slate default theme and it also doesn't reload browser. It seems like the browser wasn't using any certificate from node-modules before I've tried Slate v1. I just have no idea how to force browser not to use any certificate.

Thank you.

**Comment by t-kelly** [comment]
@radmedov if you're in MacOS, try opening the Keychain Access app and deleting any cert referencing localhost?

![image](https://user-images.githubusercontent.com/4837696/49808542-ce83d480-fd2a-11e8-8ea9-8330e5056191.png)

**Comment by radmedov** [comment]
@t-kelly, I did removed it immediately I got back to Slate v0. Sorry if I wasn’t clear at the beginning. I did remove all related to the Slate v1, re-install node.

**Comment by lizsterine** [comment]
@radmedov Have you been able to fix your issue?  I just tried Slate 1 and now my Slate 0 isn't reloading either :( .

**Comment by radmedov** [comment]
@lizsterine Nope, I gave up forcing it to work. I'm using Slate v1 now for developing from scratch.
And [Parcel.js](https://medium.com/@danwebb/better-shopify-theme-development-with-parcel-js-704f17f367fc) and [LivePage Chrome extension](https://chrome.google.com/webstore/detail/livepage/pilnojpmdoofaelbinaeodfpjheijkbh) .
It's just working fine for that purpose.

**Comment by dmatora** [comment]
> You can still use Slate v0 + LivePage Extension. Just make sure to set up the poll for updates more than 1500ms.
OMG! Just tried LivePage Extension and it works with any configuration! Thank you!

**Comment by GuillermoCasanova** [comment]
Ran into this issue a well. I've ended up "solving" it by copy pasting node_modules from a .13 project into a .14 project. And I say "solving" it because I still need to hit CMD + Save a few time before it reloads with the changes. 

And this all started happening after trying 1.0.0 beta and slate 0.14.0
